python/data_extraction/export_data.py

```diff
@@ -25,6 +25,17 @@
                                 content_store_url=plek.find('draft-content-store'))
 
 
+untagged_content_links_generator = content_export.content_links_generator(
+    additional_search_fields={'reject_part_of_taxonomy_tree': taxonomy_query.TaxonomyQuery().root_taxon_content_ids()},
```

Comment:
I think there is some ambiguity in the card requirements, and you may have clarified this already but I'll bring it up anyway in the hope that you can clarify it for me. The Trello description has as follows

> We should identify all in-scope content items not associated with a taxon

My understanding of this is that a taxon can be anywhere, and not just attached to the 'topic taxonomy' (i.e. the one built up here from the level_one_taxons). With the current implementation, we might have a problem that the World taxons would be included, which might not be what we're after.

Reply:
Good point. I thought this would be correct - I'll ask for some clarification

Reply:
Fixed this now.